stock_request/tests/test_stock_request.py

```diff
@@ -157,17 +157,20 @@ def test_defaults_order(self):
             order.location_id, self.warehouse.lot_stock_id)
 
     def test_onchanges_order(self):
+        expected_date = fields.Date.today()
```

Comment:
```suggestion
        expected_date = fields.Datetime.now()
```

Reply:
There are more tests that relay on today date. As we only need to ensure that date expected is the same in SRO and SR it's fine.

Reply:
@hveficent I am agree with your point. But `expected_date` field type is `datetime`.